src/domains/functions/function_admin.tf

```diff
@@ -299,6 +299,96 @@ module "function_admin_staging_slot" {
   tags = var.tags
 }
 
+// ----------------------------------------------------
+// Alerts
+// ----------------------------------------------------
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "alert_failed_delete_procedure" {
+  enabled             = true
+  name                = "[IO-AUTH | ${module.function_admin.name}] Found one or more failed DELETE procedures"
+  resource_group_name = azurerm_resource_group.admin_rg.name
+  scopes              = [data.azurerm_application_insights.application_insights.id]
+  description         = <<EOT
+    Found one or more failed DELETE procedures.
+    Check ${local.function_admin.app_settings_common.FAILED_USER_DATA_PROCESSING_TABLE} table in
+    ${data.azurerm_storage_account.storage_api.name} storage account for more details
+    EOT
+
+  severity                = 1
+  auto_mitigation_enabled = false
+  location                = azurerm_resource_group.admin_rg.location
+
+  // check once every minute(evaluation_frequency)
+  // on the last minute of data(window_duration)
+  evaluation_frequency = "P1D"
+  window_duration      = "P1D"
+
+  criteria {
+    query                   = <<-QUERY
+exceptions
+| where customDimensions.name startswith "user.data.delete"
+| where customDimensions.isReplay == true
+    QUERY
+    operator                = "GreaterThanOrEqual"
+    time_aggregation_method = "Count"
+    threshold               = 1
+    failing_periods {
+      minimum_failing_periods_to_trigger_alert = 1
+      number_of_evaluation_periods             = 1
+    }
+  }
+
+  # Action groups for alerts
+  action {
+    action_groups = [data.azurerm_monitor_action_group.io_auth_error_action_group.id]
+  }
+
+  tags = var.tags
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "alert_failed_download_procedure" {
+  enabled             = true
+  name                = "[IO-AUTH | ${module.function_admin.name}] Found one or more failed DOWNLOAD procedures"
+  resource_group_name = azurerm_resource_group.admin_rg.name
+  scopes              = [data.azurerm_application_insights.application_insights.id]
+  description         = <<EOT
+    Found one or more failed DOWNLOAD procedures.
+    Check ${local.function_admin.app_settings_common.FAILED_USER_DATA_PROCESSING_TABLE} table in
+    ${data.azurerm_storage_account.storage_api.name} storage account for more details
+    EOT
+
+  severity                = 1
+  auto_mitigation_enabled = false
+  location                = azurerm_resource_group.admin_rg.location
+
+  // check once every minute(evaluation_frequency)
+  // on the last minute of data(window_duration)
+  evaluation_frequency = "P1D"
+  window_duration      = "P1D"
+
+  criteria {
+    query                   = <<-QUERY
+exceptions
+| where customDimensions.name startswith "user.data.download"
+| where customDimensions.isReplay == true
```

Comment:
```suggestion
| where customDimensions.isReplay == false
```

Reply:
I intentionally put true to exclude false positives like, for example, whenever the orchestrator files a retry and that goes fine without any error.

Reply:
We never raise an exception in this scenario.
And what if we add a new activity after this log? We'll have three logs, two of them with `isReplay=true`

Reply:
you are right. however I removed the filter to align the upstream implementation in https://github.com/pagopa/io-infra/pull/1334/commits/1636a27919134d045b53b406175c4f0c4adff880